http-api/src/main/java/io/avaje/http/api/HttpStatus.java

```diff
@@ -0,0 +1,84 @@
+package io.avaje.http.api;
+
+public enum HttpStatus {
```

Comment:
idk, most microframeworks provide their own HTTP status enum

Reply:
But how would they use it on an avaje annotation?

Currently we use an int meaning folks need to know the http status codes which is maybe OK but in theory this helps them.  Is there a better way?

Reply:
Idk, I like using int

Reply:
OK, we can stick with int.

Reply:
I have the thought of adding a java comment in the generated code [where the generated code sets the status] that translates the status code int into the reason description (for the purpose of helping the person reading the generated code who may not know the status code being used).

That way we use int, but at least the generated code might be more readable for devs reading the [generated] code later.

Reply:
sounds good to me